cookbooks/megam_elasticsearch/templates/default/hosts.erb

```diff
@@ -0,0 +1 @@
+<%=node['ipaddress']%>    <%=node['hostname']%>
```

Comment:
remove this file
We do this in megam_preinstall
Refer https://github.com/megamsys/chef-repo/blob/master/cookbooks/megam_preinstall/recipes/default.rb